README.md

```diff
@@ -19,6 +19,9 @@ Work in progress to adapt device specific and customer-specific out-of-tree conf
 While in build directory, run
 
     $ nix-build -I nixpkgs=nixpkgs -I spectrum-config=build-configurations/<config-name>.nix build-configurations/release/live/
+Currently available config files are:
+* imx8qm-config.nix
+* x86-nuc-config.nix
```

Comment:
Let's create a markdown table out of these with copy-pasteable build commands for particular build configuration instead of <config-name>-template. 
For user friendliness - it's more easy to use copy-pasteable build command than go through the templated line and make correct replacement.

Reply:
Addressed